src/pages/code-of-conduct.js

```diff
@@ -0,0 +1,213 @@
+              immediately. You may report the violation in a number of ways:
+            </p>
+
+            <ol className="list-disc pl-6 mb-4">
+              <li>By finding an organizer and reporting in person</li>
+              <li>
+                Contacting in organizer directly, or corporately, via email:
+                <ol className="list-disc pl-6 mb-4 mt-2">
+                  <li>
+                    Seth Alexander:{' '}
+                    <a href="mailto:setha@hey.com">setha@hey.com</a>
+                  </li>
+                  <li>
+                    Laura Pinell:{' '}
+                    <a href="mailto:laura@uxbridge.io">laura@uxbridge.io</a>
+                  </li>
+                </ol>
+              </li>
+              <li>
+                All reports will be kept confidential.** In some cases we may
+                determine that a public statement will need to be made. In those
+                cases, the identities of all victims, reporters, and the accused
+                will remain confidential unless those individuals instruct us
+                otherwise.
+              </li>
+            </ol>
+
+            <p className="mb-5">
+              Organizers will be happy to help members contact local law
+              enforcement, provide escorts, or otherwise assist any attendee to
+              feel safe for the duration of any event. We value your presence.
+            </p>
+
+            <p className="mb-5">When making a report, please include:</p>
+
+            <ol className="list-disc pl-6 mb-4">
+              <li>Your contact info for follow-up contact.</li>
+              <li>
+                Names (legal, nicknames, or pseudonyms) of any individuals
+                involved.
+              </li>
+              <li>
+                If there were other witnesses besides you, please try to include
+                them as well.
+              </li>
+              <li>
+                When and where the incident occurred. Please be as specific as
+                possible.
+              </li>
+              <li>Your account of what occurred.</li>
+              <li>
+                If there is a publicly available record (e.g. an email archive
+                or a public Slack log) please include a link.
+              </li>
+              <li>Any extra context you believe existed for the incident.</li>
+              <li>If you believe this incident is ongoing.</li>
+              <li>Any other information you believe we should have.</li>
+            </ol>
+
+            {/* ------------------- Appeals ------------------- */}
+
+            <h3>Appeals</h3>
+
+            <p className="mb-5">
+              Only permanent actions (such as bans) may be appealed. To appeal a
+              decision, please contact an organizer listed above with your
+              appeal and freeCodeCamp Nashville will review the case. License
```

Comment:
Request to remove "License" from the end of this line. It was also removed from the code of conduct we sourced this from.

Reply:
Removed the text.